app/views/cooking_sessions/index.html.erb

```diff
@@ -1,24 +1,41 @@
-<h1>Check out these meals your neighbors are cooking!</h1>
-<br>
+<div class="container mt-3">
+  <%= simple_form_for :search, url: cooking_sessions_path, method: :get do |f| %>
+    <%= f.input :query,
+      label: "Location:",
+      input_html: { value: last_location.address }
+    %>
+    <%= f.input :max_distance,
+      label: "Meals available in:",
```

Comment:
suggestion for wording:
meals available **within**
or just
meals within

Reply:
Modification applied (changed to meals within)